src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -1023,17 +1025,23 @@ g.test('texture_usage')
       .unless(({ usage0, usage1 }) => {
         return !IsValidTextureUsageCombination(usage0 | usage1);
       })
+      .combine('depthOrArrayLayers', [1, 2])
+      .unless(({ dimension, depthOrArrayLayers }) => {
+        return dimension === '1d' && depthOrArrayLayers !== 1;
+      })
+      .combine('mipLevelCount', [1, 2])
```

Comment:
This makes the whole test roughly 4x as long. Technically you could add these extra cases only when usage has TRANSIENT_ATTACHMENT, but that's hard to do.

I think it would be better to add a separate test that just these validation rules. They shouldn't interact with most of the parameters so I don't think they need to be tested in combination.

(note to self, definitely related to why we have too many tests. Both doing conditional expansions and adding new tests is more annoying than just adding combinatorics.)